tools/MultiProjectValidator/project.json

```diff
@@ -5,7 +5,7 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "Microsoft.NETCore.App": "1.0.0-rc3-*",
+    "Microsoft.NETCore.App": "1.0.0-rc2-*",
```

Comment:
This shouldn't matter right? The tool just needs to run. And I'm removing it from this repo entirely in a PR coming today.